tests/Integration/Behaviour/Features/Context/Domain/AbstractDomainFeatureContext.php

```diff
@@ -28,18 +28,23 @@
 
 use Behat\Behat\Context\Context;
 use Behat\Testwork\Hook\Scope\BeforeSuiteScope;
+use Exception;
 use Language;
 use ObjectModel;
 use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
 use Psr\Container\ContainerInterface;
 use RuntimeException;
+use Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
 use Tests\Integration\Behaviour\Features\Context\CommonFeatureContext;
 use Tests\Integration\Behaviour\Features\Context\SharedStorage;
 
+require_once __DIR__ .'/../../../../../../vendor/phpunit/phpunit/src/Framework/Assert/Functions.php';
```

Comment:
@rokaszygmantas @matks it would be nice to phpunit assertions instead of ifs and throwing exceptions. Behat works quite nice with phpunit. Just would like to get confirmation from you guys that is a good way to go? :)

See below how I use it with:

```
        assertSame(
            $customerId,
            $customerIdReceived,
            sprintf('Expected customer with id "%s" but received "%s', $customerId, $customerIdReceived)
        );
```

Reply:
To avoid using the relative path refactored the code to use PHPUnit_Framework_Assert::assertSame